spec/takeaway_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'takeaway'
+
+describe TakeAway do
+
+  subject(:takeaway) {described_class.new()}
+  # let(:menu) { double(menu:, :pizza => 10 }
+
+  context 'Customer' do
+    it 'can see the menu' do
+      expect(takeaway).to respond_to :order
+        end
```

Comment:
Lint/BlockAlignment: `end` at 11, 8 is not aligned with `it 'can see the menu' do` at 9, 4.